graphql/schema/types/task.graphql

```diff
@@ -109,6 +109,7 @@ type Task {
   spawnHostLink: String
   startTime: Time
   status: String!
+  tags: [String!]
```

Comment:
referencing our own guidelines, I would suggest:
```suggestion
  tags: [String!]!
```

Reply:
Thanks for the callout!